### PR TITLE
Aderyn Start available as a button at the bottom right

### PR DESCRIPTION
**Pull request by TilakMaddy**
image

### CODE REVIEW

File: src/extension.ts [hunk 1]
```diff
@@ -75,6 +79,10 @@ export function activate(context: vscode.ExtensionContext) {
 
     context.subscriptions.push(runCommand);
     context.subscriptions.push(solidityWatcher);
+
+    item.text = "$(beaker) Aderyn Start";
```
Comment (TilakMaddy) [inline src/extension.ts:83]: You can substitute "beaker" with any name available here https://github.com/microsoft/vscode-codicons/tree/main/src/icons

Comment (alexroan) [inline src/extension.ts:83]: In debug mode, the extension doesn't seem to highlight the code diagnostics. I'm not sure why.

**Comment by alexroan** [comment]
Implemented separately on main branch.

**Comment by TilakMaddy** [comment]
> Implemented separately on main branch.

Did it work ??

**Pull request closed without merging**